src/components/pages/search.js

```diff
@@ -58,13 +82,29 @@ function Search({ initialState }) {
 
   return (
     <Page headerActions={[]}>
+      <h1>{`${t('searchResultPrefix')}: ${urls.decodeUrl(query.query)}`} </h1>
+
+      <div className="Search-searchSelectContainer">
+        <Form.Item label={t('refineSearch')} >
+          <Select
+            mode="multiple"
+            tokenSeparators={[' ']}
+            value={selectedTags}
+            onChange={selectedValues => handleTagsChanged(selectedValues)}
+            placeholder={searchPlaceholder()}
+            options={Array.from(allTags).map(tag => ({ value: tag, key: tag }))}
```

Comment:
I think it's maybe better to extract this in a variable and have less logic in the HTML.

Reply:
Done